samples/sampleINT8API/README.md

```diff
@@ -56,10 +56,11 @@ Specifically, this sample performs the following steps:
 
 3.  Optionally and for debugging purposes, the following flag configures the builder to choose type conforming layer implementation, if one exists.
 
-	For example, in the case of `DataType::kINT8`, types are requested by `setInt8Mode(true)`. Setting this flag ensures that only the conformant layer implementation (with `kINT8` input and output types), are chosen even if a high performance non-conformat implementation is available. If no conformant layer exists, TensorRT will choose a non-conformant layer if available regardless of the setting for this flag.
-
 	`builder->setStrictTypeConstraints(true);`
 
+	Setting `setStrictTypeConstraints(true)` together with the builder flag `setFlag(BuilderFlag::kINT8)` ensures that only the conformant layer implementation (with `kINT8` input and output types), are chosen even if a high performance non-conformat implementation is available. If no conformant layer exists, TensorRT will choose a non-conformant layer if available regardless of the setting for this flag.
```

Comment:
Thanks, just some minor grammar corrections

```suggestion
	Setting `setStrictTypeConstraints(true)` together with the builder flag `setFlag(BuilderFlag::kINT8)` ensures that only the conformant layer implementation (with `kINT8` input and output types) is chosen even if a high performance non-conformant implementation is available. If no conformant layer exists, TensorRT will choose a non-conformant layer if available regardless of the setting for this flag.
```

Reply:
@kevinch-nv good catch, changed as proposed and removed a superfluous blank line.